claragenomics/dl4atac/dataset.py

```diff
@@ -121,12 +122,13 @@ def _get_generator(self):
                 sys.stdout.flush()
             rec = self.fh_data[file_id][local_idx - self.fh_indices[file_id][0]]
             sys.stdout.flush()
-            if len(rec.shape) == 1:
+            idx = yield {'idx':idx, 'x':rec}
+            #if len(rec.shape) == 1:
                 # When no labels, return just the input data
-                idx = yield {'idx':idx, 'x':rec}
-            else:
+            #    idx = yield {'idx':idx, 'x':rec}
+            #else:
                 # Return 4 items -- IDX (for saving/tracing), input data, upsampled data, peaks/classifications
-                idx = yield {'idx':idx, 'x':rec[:,0], 'y_reg':rec[:,1], 'y_cla':rec[:,2]}
+                #idx = yield {'idx':idx, 'x':rec[:,0], 'y_reg':rec[:,1], 'y_cla':rec[:,2]}
 
 """
 # Is this even used?
```

Comment:
should we delete this whole part ? Is it functional at all ?

Reply:
I don't think so, but I'd like @tijyojwad to check and confirm.

Reply:
I deleted the commented lines. Have not deleted the DatasetEval section.

Reply:
I think it's safe to delete the DatasetEval as well. It's not being used anywhere in the code currently. @tijyojwad to confirm if I'm missing something.

Reply:
Yeah I think okay to delete dataset eval

Reply:
deleted.